src/mixins/raster-layer-line-mixin.js

```diff
@@ -364,7 +363,13 @@ export default function rasterLayerLineMixin(_layer) {
     let sql
     if (isContourType(state)) {
       validateContourState(state)
-      sql = buildContourSQL(state, mapBounds)
+      const filterTransforms = _layer
```

Comment:
so, is this an attempt to replicate the `transform` component of the `parser.writeSQL` call below? It's hard for me to say because I'm unfamiliar with the `data-layer` module, but it seems on the surface that this is simplifies the transform parsing that the `parser` object does. Is that fair to say? I'm just wondering how closely this matches the transform parse and expression build in that `data-layer` module, and if it doesn't very closely match, I wonder if we could leverage just calling the `transform` related component here.